### ISSUE TITLE
ARIA Grid: Standard table navigation commands don't work when navigating by cell.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3742]
**Reported by bgaraventa on 2014-01-02 23:32**
When an ARIA Grid construct uses all requisite ARIA roles and states to convey an ARIA Grid, such as aria-owns and proper role nesting, it is not possible to use standard table navigation commands such as Alt+Control+Left/Up/Right/Down to navigate by cell within the Virtual Buffer.

NVDA gets confused and cannot detect that it is navigating within a table.

This occurs in the latest version of Firefox.

Live demo:
http://whatsock.com/test/demo/Coding%20Arena/ARIA%20Data%20Grids/ARIA%20Data%20Grid%20(Dynamic)/demo.htm

**Comment by bhavyashah** [comment]
I believe work has been done recently to improve support for ARIA grids. Does that encompass this ticket too? If so, I suggest closing.

**Comment by ehollig** [comment]
I believe this is now fixed in Firefox. @jcsteh, any further clarification?

**Comment by jcsteh** [comment]
Firefox previously didn't support aria-owns fully, which was probably the cause of this. It does now.

@ehollig, did you re-test the test case(s) here? If not, this needs to be re-tested to confirm whether the issue is fixed.

**Comment by ehollig** [comment]
Tested both attachments and the example provided in the original post with the latest NVDA and Firefox 54 and was able to navigate the tables using the table commands. Closing as worksforme
